packages/tour-kit/src/components/tour-kit-frame.tsx

```diff
@@ -171,29 +172,18 @@ const TourKitFrame: React.FunctionComponent< Props > = ( { config } ) => {
 					{ showArrowIndicator() && (
 						<div className="tour-kit-frame__arrow" data-popper-arrow { ...arrowPositionProps } />
 					) }
-					{ ! isMinimized ? (
-						<>
-							{ config.renderers.tourStep( {
-								steps: config.steps,
-								currentStepIndex,
-								onDismiss: handleDismiss,
-								onNext: handleNextStepProgression,
-								onPrevious: handlePreviousStepProgression,
-								onMinimize: handleMinimize,
-								setInitialFocusedElement,
-								onGoToStep: handleGoToStep,
-							} ) }
-						</>
-					) : (
-						<>
-							{ config.renderers.tourMinimized( {
-								steps: config.steps,
-								currentStepIndex,
-								onMaximize: handleMaximize,
-								onDismiss: handleDismiss,
-							} ) }
-						</>
```

Comment:
We should only refactor the `tourStep` render prop above into `TourKitStep`, not `tourMinimized` (as this is not really a step currently, but another view). So let's leave that intact for now (or unless you want to create another component and call it `TourKitMinimized` 🤷‍♂️ )